98Installation/sec00virtualbox.md

```diff
@@ -8,22 +8,23 @@ title: VirtualBox
 
 We will be running a linux virtual machine using VirtualBox software on either your laptop, or if you do not have 
 one on one of the Desktop@UCL machines in Physics Lab 1. If you have a laptop this is the preferred option as it 
-will mean you will be able to work on the homework problems and coursework without needing access to a cluster room. 
-If you don’t have a laptop then that is fine as well as the whole course can be completed using the Desktop@UCL machines.
+will mean you will be able to work on the homework problems and coursework without needing access to a cluster room. If you don’t have a laptop then that is fine as well as the whole course can be completed using the Desktop@UCL machines.
+
+Download and install on your laptop:
+* VirtualBox from [virtualbox.org](https://www.virtualbox.org/wiki/Downloads) 
+* The VirtualBox Extension Pack (from same page, make sure the versions match 
 
-Download and install on your laptop VirtualBox 5.2.36 and the extension pack from [virtualbox.org](https://www.virtualbox.org/wiki/Download_Old_Builds_5_2). 
 If you are using one of the Desktop@UCL machines then VirtualBox is already installed.  
 
 ### Creating the VM and installing Ubuntu
 
-Download the Ubuntu 18.04.3 LTS from https://ubuntu.com/download/desktop and save the .iso file somewhere locally on your 
-laptop or if you are using one of the Desktop@UCL machines someone in your home folder.
+Download the Ubuntu 18.04.3 LTS from https://ubuntu.com/download/desktop and save the .iso file somewhere locally on your laptop or if you are using one of the Desktop@UCL machines someone in your home folder.
 
 Then open VirtualBox and create a new VM of Type: Linux and Version: Ubuntu 64 bit. 2 GB ram is sufficient. 
 
 Choose `Create a virtual hard disk now` and select it to be of type VirtualBox Disk Image (VDI) that is Dynamically allocated. Between 10 GB and 20 GB storage is sufficient. 
-If you are using a Desktop@UCL machine then it is important that you save the virtual machine .vdi file to your network drive in 
-a folder in `Home (N:)` so that you can access the virtual machine from different Desktop@UCL machines. 
+
+If you are using a Desktop@UCL machine then it is important that you save the virtual machine .vdi file to your network drive in a folder in `Home (N:)` so that you can access the virtual machine from different Desktop@UCL machines. 
```

Comment:
```suggestion
If you are using a Desktop@UCL machine then it is important that you save the virtual machine `.vdi` file to your network drive in a folder in `Home (N:)` so that you can access the virtual machine from different Desktop@UCL machines. 
```